content/resources/help-centre/software-tips/path.md

```diff
@@ -0,0 +1,96 @@
+full path! 
+
+```ps
+> AnalysisPrograms.exe
+```
+
+## How to check the PATH variable
+
+We can quickly check if a program has been successfully added to our `PATH` or
+not. Open up a shell (such as PowerShell on Windows, or terminal on
+Linux/macOS), and type a command for the program we are interested in testing. 
+
+```ps
+> FFmpeg
+```
+
+If the command successfully returns some kind of output about the package, then
+it is available on your `PATH`.  
+
+## Adding to PATH
+
+If you use a [package manager]({{% ref "package-manager.md" %}}) such as
+[chocolatey](https://chocolatey.org/) on Windows, [homebrew](https://brew.sh/)
+on macOS or [snap](https://snapcraft.io/) on Linux, then packages you install
+with those commands should be automatically added to your `PATH` variable. This
+saves you having to manually change the `PATH` variable. The automatic installer
+for [AP](https://ap.qut.ecoacoustics.info/basics/installing.html?tabs=windows)
+will also automatically add AP to `PATH` for you.
+
+If you need to add a program to the `PATH` manually, you can follow these steps:
+
+{{% tabs "Add to PATH" %}}
+{{% tab "Windows" %}}
+1. Find where the program is installed on your computer. This will be a
+   directory (folder) where the `.exe` resides.
+2. Open your System Environment Variables. You can type some of Edit environment variables for your account in the Start Menu search box to find the settings.
+3. Choose Environment Variables... in the windows that popped up
+4. In the user variables section, find the Path variable, select it, and then hit the Edit button
+5. Add the directory from step 1 to the end
+  - Ensure a semi-colon (;) delimits the new directory from the previous ones, if you're using an older version of Windows
+6. Then click OK or close all windows.
+7. You will have to restart any programs for which you want to see the new value
+{{% /tab %}}
+{{% tab "macOS" %}}
+1. Find where the program is installed on your computer. This will be a
+   directory (folder) where the .exe resides.
+2. Open or create your ~/.profile file
+3. Add the following line to the end:
+
+`PATH=$PATH:<REPLACE-ME>`
+
+where you replace the `<REPLACE-ME>` with the directory from step 1.
+
+4. Close and save the file
+5. Run the same command in your current shell (PATH=$PATH:<REPLACE-ME>) to see the change take effect immediately (or restart your shell).
+{{% /tab %}}
+{{% tab "Linux" %}}
+1. Find where the program is installed on your computer. This will be a
+   directory (folder) where the .exe resides.
+2. Open or create your ~/.profile file
+3. Add the following line to the end:
+
+`PATH=$PATH:<REPLACE-ME>`
+
+where you replace the `<REPLACE-ME>` with the directory from step 1.
+
+4. Close and save the file
+5. Run the same command in your current shell (PATH=$PATH:<REPLACE-ME>) to see the change take effect immediately (or restart your shell).
+{{% /tab %}}
+{{% /tabs %}}
```

Comment:
so fun fact: there are system wide environment files you can configure for linux (which would be the same as your windows example)
![image](https://user-images.githubusercontent.com/1452968/199008210-778f75b1-8c5c-4bd7-8249-4c6ce8a720e8.png)


and in powershell you can do `$env:Path=$env:Path;<REPLACE-ME>` in your powershell profile as well!